src/helpers/daily-data-providers/apple-health-sleep.ts

```diff
@@ -116,4 +123,20 @@ export function asleepTime(startDate: Date, endDate: Date) {
 
 export function inBedTime(startDate: Date, endDate: Date) {
 	return coreSleep(startDate, endDate, "InBed");
+}
+
+export function asleepCoreTime(startDate: Date, endDate: Date) {
+	return coreSleep(startDate, endDate, "AsleepCore");
+}
+
+export function asleepRemTime(startDate: Date, endDate: Date) {
+	return coreSleep(startDate, endDate, "AsleepREM");
+}
+
+export function asleepDeepTime(startDate: Date, endDate: Date) {
+	return coreSleep(startDate, endDate, "AsleepDeep");
+}
+
+export function asleepAnyTime(startDate: Date, endDate: Date) {
```

Comment:
Let's remove this one and instead we should update "asleepTime" function above to include ```["Asleep", "AsleepCore", "AsleepREM", "AsleepDeep" ]